audmath/core/api.py

```diff
@@ -86,6 +88,207 @@ def db(
     return x
 
 
+def duration_in_seconds(
+        duration: typing.Union[float, int, str, np.timedelta64],
+        sampling_rate: int = None,
```

Comment:
```suggestion
        sampling_rate: typing.Union[float, int] = None,
```

At least the following works as expected:

```python
>>> audmath.duration_in_seconds('3', sampling_rate=1.5)
2.0
```